CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Add your own contributions to the next release on the line below this, please include your name too. Please don't set a new version if you are the first to make the section for `master`.
 
 * Improve remotes parsing for init command - [@sleekybadger](https://github.com/sleekybadger)
+* Add ability to get whole diff - [@sleekybadger](https://github.com/sleekybadger)
```

Comment:
Any chance we can expand this a bit more: `Adds ability to get the whole diff object at `git.diff` - this lets you do more fine grained checks on the raw changes.`

Then people will know how to use it